cloudify_aws/lambda_serverless/resources/permission.py

```diff
@@ -72,7 +75,7 @@ def prepare(ctx, resource_config, **_):
             utils.update_resource_id(
                 ctx.instance, resource_config['StatementId'])
         else:
-            utils.update_resource_id(ctx.instance, str(uuid4()))
+            utils.update_resource_id(ctx.instance, text_type(uuid4()))
```

Comment:
Same question. No change is necessary, just curious.

Reply:
Same as above